crypto/evp/evp_enc.c

```diff
@@ -934,40 +942,50 @@ int EVP_CIPHER_CTX_set_key_length(EVP_CIPHER_CTX *c, int keylen)
 
 int EVP_CIPHER_CTX_set_padding(EVP_CIPHER_CTX *ctx, int pad)
 {
+    int ok;
```

Comment:
missing blank line after declaration.  but you can avoid the local variable altogether.

Reply:
Ok